documentation/docs/scenarios/upload-to-transportrequest/Readme.md

```diff
@@ -0,0 +1,197 @@
+# Build an SAP Fiori Application and attach it to a transport request on an ABAP system with Jenkins
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and attaches the build result to a transport request in an ABAP system.
+
+There are in principle two technical ways how to attach a binary to an ABAP transport request. We support an upload via RFC and an upload via ODATA. Which way to use depends on the version of your ABAP system. For AS ABAP 7.50 SP08 or 7.51 SP07 or 7.52 SP03 or newer the ODATA based upload should be used, for older versions the RFC based upload needs to be used.
+
+## Prerequisites
+
+* [Docker environment](https://docs.docker.com/get-started/)
+* You have set up Project “Piper”. See [guided tour](../../../guidedtour/)
+* A transport request
+* Depending on the version of the ABAP system: Docker image for attaching binaries to transport requests via RFC available. Due to legal reasons there is no pre-build docker image. How to create the docker image is explained [here](https://github.com/SAP/devops-docker-images/tree/master/node-rfc)
```

Comment:
Is this still the place to go? Is `devops-docker-images` not deprecated?

Reply:
To my knowledge there are still some remaining images provided from that repo. Which repo is according to your view the new one which we can refer to instead?